### PR TITLE
Add spec for alert repair/remediate

### PR DESCRIPTION
**Pull request by sumitabarahmand**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by sumitabarahmand** [comment]
@bganapa  @deathly809

**Comment by bganapa** [comment]
@sarangan12 This is going thru azure stack admin api review board, please wait for sign off from @deathly809  or @bganapa  before merge

File: specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json
Comment (knithinc) [inline specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json]: Is there nothing returned as part of the response body?

Comment (deathly809) [reply specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json]: If this is a long running operation please mark it as such.  If the long running operation returns an object please modify the 200 and 202 to return that object, which MUST be an Alert object.

File: specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json
Comment (deathly809) [inline specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json]: This is not a good name.  Please change to Remediate.

File: specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json
Comment (deathly809) [inline specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json]: Please don't call the post action remediate and then describe it as repair.  This does not repair an alert.

File: specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json
Comment (deathly809) [inline specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json]: You don't need this here.  There should be a global definition of this which covers everyone.

File: specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json
Comment (sumitabarahmand) [reply specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json]: When we do the post operation, the response body is empty and the operation tracking URI is in the response headers. So yes, there is nothing returned as a part of the response body.

File: specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json
Comment (sumitabarahmand) [reply specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json]: This was the guidance given to us by our PM (Charlie). Remediate is not an approved powershell verb and the UI and powershell should use repair.

File: specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json
Comment (sumitabarahmand) [reply specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json]: As the powershell cmdlet and UI will be using repair I aligned the text in this file with that.

Comment (bganapa) [reply specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json]: The api action need not match what you need in powershell.. in any case Repair does not sound like the right term here

File: specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json
Comment (bganapa) [reply specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json]: Please add        "x-ms-long-running-operation": true

File: specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json
Comment (sumitabarahmand) [reply specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json]: The route was updated to repair.

**Comment by sarangan12** [comment]
@sumitabarahmand The semantic validator is failing. Please fix it.

**Comment by bganapa** [comment]
@sumitabarahmand Are you planning to take this in for the upcoming AZURESTACK release?

**Review by bganapa — APPROVED** [review]

File: specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json
Comment (sumitabarahmand) [reply specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json]: I updated it to bool. thanks for the catch.

**Comment by bganapa** [comment]
@shahabhijeet this should be good to merge

**Comment by bganapa** [comment]
@shahabhijeet @sarangan12 , can one of you merge this pls, we had to update the SDK PR once this is merged.. Thanks!
https://github.com/Azure/azure-sdk-for-net/pull/5000

**Review by sarangan12 — APPROVED** [review]

**Pull request merged**
